src/lib/util.js

```diff
@@ -1,5 +1,29 @@
+const os = require('os');
+const path = require('path');
+
 function isFunction(val) {
-    return val && val.apply !== undefined; //  typeof val === 'function';
+    return val && val.apply !== undefined;
+}
+
+function isConstructor(obj) {
+    if (!obj) return false;
+    if (!obj.prototype) return false;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
    if (!obj.prototype) {return false;}
```